cmake/dependencies/antares-solver.cmake

```diff
@@ -0,0 +1,64 @@
+#####################
+##  Antares solver  ##
+#####################
+message("DEPENDENCY ANTARES " ${BUILD_antares_solver})
+message("DEPENDENCY ANTARES " ${DOWNLOAD_antares_solver})
+if(BUILD_antares_solver OR DOWNLOAD_antares_solver)
+    find_package(antares-solver QUIET)
+    if (NOT antares-solver_FOUND)
+        if (DOWNLOAD_antares_solver)
+            message(${DEPS_INSTALL_DIR}/..)
+            if (MSVC)
+            message(STATUS "Will download Antares solver windows assets")
+                set(PLATFORM_TYPE installer-64bits.zip)
+            else()
+                execute_process(COMMAND bash "-c" "lsb_release -d | awk -F\"\t\" '{print $2}'" OUTPUT_VARIABLE SYSTEM_FLAVOUR)
+                string(FIND ${SYSTEM_FLAVOUR} "Ubuntu" UBUNTU_FOUND)
+                if (UBUNTU_FOUND EQUAL -1)
+                    message(STATUS "Will download Antares solver CentOS assets")
+                    set(PLATFORM_TYPE CentOS-7.9.2009-ortools-xpress.tar.gz)
+                else()
+                    message(STATUS "Will download Antares solver Ubuntu assets")
+                    set(PLATFORM_TYPE ubuntu-20.04.tar.gz)
+                endif()
+            endif()
+            execute_process(COMMAND rm -rf antares-${ANTARES_VERSION_TAG}-${PLATFORM_TYPE}) #Remove old version, just in case
+            execute_process(COMMAND
+                    wget https://github.com/AntaresSimulatorTeam/Antares_Simulator/releases/download/v${ANTARES_VERSION_TAG}/antares-${ANTARES_VERSION_TAG}-${PLATFORM_TYPE}
+                    )
```

Comment:
- `wget -O output` overwrites `output`
- Naming is not uniform rte-antares (Windows), antares-*-Ubuntu, antares-*-CentOS, etc. It seems that l. 25 only works in Ubuntu & CentOS.

Reply:
Indeed didn't see it